axes/conf.py

```diff
@@ -118,3 +118,12 @@
 
 # set CORS allowed origins when calling authentication over ajax
 settings.AXES_ALLOWED_CORS_ORIGINS = getattr(settings, "AXES_ALLOWED_CORS_ORIGINS", "*")
+
+# set the list of sensitive parameters to cleanse from get/post data before logging
+settings.AXES_SENSITIVE_PARAMETERS = getattr(
+    settings,
+    "AXES_SENSITIVE_PARAMETERS",
+    [
+        "password",
```

Comment:
Should other sensitive parameters be concatenated to `["password"]` by default? If a user defines a list that overrides this and doesn't have `"password"` included as an element then it will be visible in the logs.

Reply:
This is a good point.  I've consolidated the handling of ``AXES_SENSITIVE_PARAMETERS``, ``AXES_PASSWORD_FORM_FIELD`` and also hard-coded a case for ``'password'`` to match the existing functionality.